### PR TITLE
[ENH] - Update Shuffle Circular-Bins

### PR DESCRIPTION
**Pull request by maestapereira**
Updating `shuffle_bins` to prevent broken case scenarios.
Addressing #107.

What's changed:
* Units: milliseconds -> seconds
* Argument default: `bin_width_range=[50, 2000]` -> `bin_width_range=[.5, 7]`

What's added:
* More description for `bin_width_range` argument in the Notes of the docstring.
* New spikes to `tspikes()` (`conftest.py`) due to the new maximum in `bin_width_range=[.5, 7]` default.
* New ISIs to `test_compute_isis(tspikes)` to accommodate new spikes in `tspikes`.

### CODE REVIEW

File: spiketools/stats/shuffle.py [hunk 1]
```diff
@@ -75,15 +75,15 @@ def shuffle_isis(spikes, n_shuffles=1000):
...
     """Shuffle data with a circular shuffle of the spike train.
 
     Parameters
     ----------
     spikes : 1d array
-        Spike times.
-    bin_width_range : list of int
-        Range of bin widths to shuffle by.
+        Spike times, in seconds.
+    bin_width_range : list of [int, int], optional, default : [.5, 7]
```
Comment (TomDonoghue) [inline spiketools/stats/shuffle.py:85]: list of [float, float]

File: spiketools/stats/shuffle.py [hunk 2]
```diff
@@ -75,15 +75,15 @@ def shuffle_isis(spikes, n_shuffles=1000):
     return shuffled_spikes
 
 
-def shuffle_bins(spikes, bin_width_range=(50, 2000), n_shuffles=1000):
+def shuffle_bins(spikes, bin_width_range=[.5, 7], n_shuffles=1000):
     """Shuffle data with a circular shuffle of the spike train.
```
Comment (TomDonoghue) [inline spiketools/stats/shuffle.py:79]: This description (from before) seems to be not quite right - I think it was copied from shuffle_circular, but it should really say something about being a binned shuffling approach.

**Review by TomDonoghue — CHANGES_REQUESTED** [review]

**Comment by TomDonoghue** [comment]
I left a couple small comments - otherwise this looks good to me!

File: spiketools/stats/shuffle.py (hunk 1 above)
Comment (maestapereira) [reply spiketools/stats/shuffle.py:85]: Good catch!

**Review by TomDonoghue — APPROVED** [review]

**Pull request merged**